src/main.cc

```diff
@@ -620,41 +620,117 @@ print_version(void)
  * Yield Tarantool help message to the given stream.
  */
 static void
-print_help(FILE *stream, const char *program)
+print_help(FILE *stream)
 {
-	static const char help_msg[] = "Tarantool %s\n\n"
-		"Run Tarantool instance:\n\n"
-		"  %s --name example-name --config ./config.yaml\n\n"
-		"Connect to an instance:\n\n"
-		"  tt connect <uri>\n\n"
-		"Execute Lua script with bundled LuaJIT:\n\n"
-		"  %s old-style-script.lua [OPTIONS] [ARGS]\n\n"
-		"Run interactive LuaJIT interpreter:\n\n"
-		"  %s -i\n\n"
-		"Usage:\n\n"
-		"  %s [OPTIONS] [SCRIPT [ARGS]]\n\n"
-		"Options:\n\n"
-		" -h, --help             display this help and exit\n"
-		" --help-env-list        display env variables taken into account\n"
-		" --force-recovery       enable force-recovery\n"
-		" -v, --version          print program version and exit\n"
-		" -c, --config PATH      set a path to yaml config file as 'PATH'\n"
-		" -n, --name INSTANCE    set an instance name as 'INSTANCE'\n"
-		" -i                     enter interactive mode\n"
-		" -e EXPR                execute string 'EXPR'\n"
-		" -l NAME                require library 'NAME'\n"
-		" -j cmd                 perform LuaJIT control command\n"
-		" -b ...                 save or list bytecode\n"
-		" -d                     activate debugging session for script\n"
-		" --failover             run a failover coordinator (Enterprise Edition)\n"
-		" --                     stop handling options\n"
-		" -                      execute stdin and stop handling options\n"
-		"\n"
-		"Please visit project home page at https://tarantool.org\n"
-		"to see online documentation, submit bugs or contribute a patch.\n"
-		;
-	fprintf(stream, help_msg, tarantool_version(),
-		program, program, program, program);
+	static const char help_msg[] = "Tarantool %s\n"
+		"\n"
+		"----------------------------------------------------------\n"
+		"\n"
+		"The recommended way to work with Tarantool is to use tt\n"
```

Comment:
Minor: It's worth to add the link to `tt` documentation in case more info is needed.

Reply:
It looks as a good idea on the first glance, but if I'll do it, it looks strange that we link `tt`'s documentation directly, but not `tarantool`'s documentation: CLI and configuration options, deploy guides and so on. https://tarantool.io contains all the topics and we link it on the bottom of the help message.

I think that the actual problem is that `tt`'s documentation is burried too deep into the website structure and we should place it to a more visible place on the website.

I would leave the help message it as is for now, because I don't know how to make the links informative and consistent without changes on the website.

Reply:
I changed my mind after discussions with Igor and the documentation team. Added the link in PR #9377.